<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TabS</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ressource.qrc">
    <normaloff>:/images/TabS</normaloff>:/images/TabS</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="topMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="MyTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuRe_Open">
     <property name="title">
      <string>Re-Open</string>
     </property>
     <widget class="QMenu" name="menuBy_artist">
      <property name="title">
       <string>By artist</string>
      </property>
      <addaction name="separator"/>
     </widget>
     <widget class="QMenu" name="menuBy_date">
      <property name="title">
       <string>By date</string>
      </property>
      <addaction name="separator"/>
     </widget>
     <addaction name="actionOpen_previous_tabs"/>
     <addaction name="actionEdit_List"/>
     <addaction name="actionClear"/>
     <addaction name="separator"/>
     <addaction name="menuBy_artist"/>
     <addaction name="menuBy_date"/>
     <addaction name="separator"/>
    </widget>
    <addaction name="actionNew"/>
    <addaction name="actionAssist"/>
    <addaction name="separator"/>
    <addaction name="menuRe_Open"/>
    <addaction name="actionOpen"/>
    <addaction name="actionOpen_Folder"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="actionExport_PDF"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
    <addaction name="actionClose_All"/>
    <addaction name="separator"/>
    <addaction name="actionPreview"/>
    <addaction name="actionPrint"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionSelect_All"/>
    <addaction name="actionFind"/>
    <addaction name="actionFind_next"/>
    <addaction name="separator"/>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuDisplay">
    <property name="title">
     <string>Display</string>
    </property>
    <addaction name="actionFull_Screen"/>
    <addaction name="actionEdit_mode"/>
    <addaction name="actionRead_only_mode"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionChords_manager"/>
    <addaction name="separator"/>
    <addaction name="actionSearch_lyrics"/>
    <addaction name="actionSearch_XTA"/>
    <addaction name="separator"/>
    <addaction name="actionDownload_XTA"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>?</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuTabs">
    <property name="title">
     <string>Tabs</string>
    </property>
    <addaction name="actionPrevious"/>
    <addaction name="actionNext"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuDisplay"/>
   <addaction name="menuTabs"/>
   <addaction name="menuTools"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="ressource.qrc">
     <normaloff>:/images/open</normaloff>:/images/open</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="ressource.qrc">
     <normaloff>:/images/new</normaloff>:/images/new</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionAssist">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Assist</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+N</string>
   </property>
  </action>
  <action name="actionOpen_Folder">
   <property name="text">
    <string>Open Folder</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionEdit_List">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Edit List</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="ressource.qrc">
     <normaloff>:/images/save</normaloff>:/images/save</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="icon">
    <iconset resource="ressource.qrc">
     <normaloff>:/images/save-as</normaloff>:/images/save-as</iconset>
   </property>
   <property name="text">
    <string>Save as</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="ressource.qrc">
     <normaloff>:/images/close</normaloff>:/images/close</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionClose_All">
   <property name="text">
    <string>Close All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+W</string>
   </property>
  </action>
  <action name="actionPreview">
   <property name="icon">
    <iconset resource="ressource.qrc">
     <normaloff>:/images/print-preview</normaloff>:/images/print-preview</iconset>
   </property>
   <property name="text">
    <string>Preview</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+P</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset resource="ressource.qrc">
     <normaloff>:/images/print</normaloff>:/images/print</iconset>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="ressource.qrc">
     <normaloff>:/images/app-close</normaloff>:/images/app-close</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="ressource.qrc">
     <normaloff>:/images/undo</normaloff>:/images/undo</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="ressource.qrc">
     <normaloff>:/images/redo</normaloff>:/images/redo</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="ressource.qrc">
     <normaloff>:/images/cut</normaloff>:/images/cut</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="ressource.qrc">
     <normaloff>:/images/copy</normaloff>:/images/copy</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="ressource.qrc">
     <normaloff>:/images/paste</normaloff>:/images/paste</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionSelect_All">
   <property name="text">
    <string>Select All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Find</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionFind_next">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Find next</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="icon">
    <iconset resource="ressource.qrc">
     <normaloff>:/images/config</normaloff>:/images/config</iconset>
   </property>
   <property name="text">
    <string>Preferences</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionFull_Screen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="ressource.qrc">
     <normaloff>:/images/fullscreen</normaloff>
     <normalon>:/images/normalscreen</normalon>:/images/fullscreen</iconset>
   </property>
   <property name="text">
    <string>Full Screen</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Return</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionSearch_lyrics">
   <property name="icon">
    <iconset resource="ressource.qrc">
     <normaloff>:/images/web</normaloff>:/images/web</iconset>
   </property>
   <property name="text">
    <string>Search lyrics</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionSearch_XTA">
   <property name="icon">
    <iconset resource="ressource.qrc">
     <normaloff>:/images/web</normaloff>:/images/web</iconset>
   </property>
   <property name="text">
    <string>Search XTA</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionOpen_previous_tabs">
   <property name="text">
    <string>Open previous tabs</string>
   </property>
  </action>
  <action name="actionChords_manager">
   <property name="icon">
    <iconset resource="ressource.qrc">
     <normaloff>:/images/chord</normaloff>:/images/chord</iconset>
   </property>
   <property name="text">
    <string>Chords manager</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionEdit_mode">
   <property name="icon">
    <iconset resource="ressource.qrc">
     <normaloff>:/images/setup</normaloff>:/images/setup</iconset>
   </property>
   <property name="text">
    <string>Edit mode</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionRead_only_mode">
   <property name="icon">
    <iconset resource="ressource.qrc">
     <normaloff>:/images/file-icon</normaloff>:/images/file-icon</iconset>
   </property>
   <property name="text">
    <string>Read-only mode</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionExport_PDF">
   <property name="icon">
    <iconset resource="ressource.qrc">
     <normaloff>:/images/pdf</normaloff>:/images/pdf</iconset>
   </property>
   <property name="text">
    <string>Export PDF</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionDownload_XTA">
   <property name="icon">
    <iconset resource="ressource.qrc">
     <normaloff>:/images/web</normaloff>:/images/web</iconset>
   </property>
   <property name="text">
    <string>Download XTA</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionHelp">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="ressource.qrc">
     <normaloff>:/images/help</normaloff>:/images/help</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="ressource.qrc">
     <normaloff>:/images/info</normaloff>:/images/info</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionPrevious">
   <property name="text">
    <string>Previous</string>
   </property>
   <property name="shortcut">
    <string>Alt+Left</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionNext">
   <property name="text">
    <string>Next</string>
   </property>
   <property name="shortcut">
    <string>Alt+Right</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyTabWidget</class>
   <extends>QTabWidget</extends>
   <header location="global">mytabwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="ressource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>269</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
